tests/manager/sqlalchemy/model/test_circulationevent.py

```diff
@@ -107,84 +107,6 @@ def test_new_title(self, db: DatabaseTransactionFixture):
         # updating the dataset.
         assert 0 == event.license_pool.licenses_owned
 
-    def test_log(self, db: DatabaseTransactionFixture):
```

Comment:
This test was moved to `test_local.py` in the `test_collect_end_to_end` test since that is where most of the logic this was testing moved to.